src/Features/Core/Portable/CodeFixes/NamingStyle/NamingStyleCodeFixProvider.cs

```diff
@@ -130,13 +130,8 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputePreviewOp
 
             protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(CancellationToken cancellationToken)
             {
-                var factory = _startingSolution.Workspace.Services.GetService<ISymbolRenamedCodeActionOperationFactoryWorkspaceService>();
-                var newSolution = await _createChangedSolutionAsync(cancellationToken).ConfigureAwait(false);
-                return new CodeActionOperation[]
-                {
-                    new ApplyChangesOperation(newSolution),
-                    factory.CreateSymbolRenamedOperation(_symbol, _newName, _startingSolution, newSolution)
-                }.AsEnumerable();
+                return SpecializedCollections.SingletonEnumerable(
+                    new ApplyChangesOperation(await _createChangedSolutionAsync(cancellationToken).ConfigureAwait(false)));
```

Comment:
i don't see the purpose of overriding ComputeOperationsAsync.  Why are we not just a subclass of SolutionChangedCodeAction?  all this would be simpler.